doc/README.rst

```diff
@@ -10,13 +13,7 @@ Building Documentation
 Prerequisites
 =============
 
-Install `Docker <https://www.docker.com/>`_.
-
-Pull the docker container:
-
-.. code-block:: console
-
-    $ docker pull ghcr.io/xmos/doc_builder:latest
+Use the `xmosdoc tool <https://github.com/xmos/xmosdoc>`_ either via docker or install it into a pip environment.
```

Comment:
The xmosdoc repository is private, and the fwk_core repository is public.  Hence an external user can't follow the instructions given.  At a minimum, this ReadMe should include a note informing the reader of this problem.  Another solution is in making xmosdoc public (would we ever want to do that?).

Reply:
This is a valid issue for many of our repos. Also, I was not able to find the fwk_core documentation on our website. I am not sure how to fix this other than including the docs as part of the release (not sure we want to do that) or making sure our website has the docs. It's a question of whether we want customers to use fwk_core standalone or not.
The docs should however be part of the sln_voice release however?

I'll make a note saying for xmos users only.